src/backend/joanie/core/api/client/__init__.py

```diff
@@ -541,6 +542,58 @@ def withdraw(self, request, pk=None):  # pylint: disable=no-self-use, invalid-na
 
         return Response(status=HTTPStatus.NO_CONTENT)
 
+    # @extend_schema(
+    #     request=None,
+    #     responses={
+    #         204: OpenApiTypes.NONE,
+    #         422: serializers.ErrorResponseSerializer,
+    #     },
+    # )
+    @action(detail=True, methods=["POST"])
+    def submit_installment_payment(self, request, pk=None):  # pylint: disable=unused-argument
+        """
+        Pay a failed installment that was scheduled for a given order.
+        """
+        order = self.get_object()
+        if order.state != enums.ORDER_STATE_FAILED_PAYMENT:
+            return Response(
+                {
+                    "detail": "The order is not in failed payment state for last payment."
+                },
+                status=HTTPStatus.UNPROCESSABLE_ENTITY,
+            )
+
+        payment_backend = get_payment_backend()
+        installment = order.get_first_installment_refused()
+
+        card_id = request.data.get("card_id")
```

Comment:
In other endpoints, we call it `credit_card_id`. `card_id` is nice too my point is just to try to use the same parameter name everywhere for consistency.
```suggestion
        card_id = request.data.get("credit_card_id")
```